components/BioBar/BioBar.tsx

```diff
@@ -15,8 +15,8 @@ const BioFooter = ({ author }: Props) => {
 
   const { name, image, bio, username } = author;
   return (
-    <div className="max-w-xl px-4 mx-auto  text-gray-700 dark:text-gray-300 mt-6">
-      <div className="flex mx-2 sm:mx-6 md:mx-auto px-4 border-t-2 pt-6 border-gray-300 dark:border-gray-800">
+    <div className="max-w-xl mx-auto  text-gray-700 dark:text-gray-300 mt-6">
```

Comment:
Could we consider removing the double-space present in this line between `mx-auto` & `text-gray-700` ? 🤔 It's not a big deal that won't break anything, but it is a nice standard to adhere to 😅

Reply:
Nice catch! Eagle eyes 🦅